terraform-sentinel/step2.md

```diff
@@ -0,0 +1,79 @@
+Now that you have created a policy, you will add some additional restrictions. 
+
+In your policy file`terraform-sentinel/restrict-s3-buckets.sentinel`{{open}}, you will first add a print statement.
+
+# Create a print statement for debugging
+
+Your policy plan collection contains a lot of data. To see what data you are giving to your policy, create a print statement below the ending bracket your filter statement.
+
+```
+print(s3_buckets)
+```{{copy}}
+
+Run your Sentinel CLI apply again to see what data your plan contains.
```

Comment:
Is there anything we want to highlight here? Can we make the output a little more human readable?

Reply:
I've added a little jq command but it might just distract?